buildroot/s390.config

```diff
@@ -0,0 +1,11 @@
+BR2_s390x=y
+BR2_OPTIMIZE_3=y
+BR2_KERNEL_HEADERS_4_4=y
+BR2_GCC_VERSION_8_X=y
+BR2_TARGET_GENERIC_ROOT_PASSWD="root"
+BR2_TARGET_GENERIC_GETTY_PORT="ttyAMA0"
```

Comment:
is this port correct? I copied this from a different arch's config, so I suppose it's likely incorrect.

Reply:
I would suggest using `configs/qemu_s390x_defconfig` as a template or are there reasons for not doing it ?